0.preprocess-sites/4.image-and-segmentation-qc.py

```diff
@@ -291,42 +347,40 @@
 # Any point too high or too low may have focus issues
 pll_col_prefix = "ImageQuality_PowerLogLogSlope_"
 PLLS_df_cols = image_meta_col_list.copy()
-PLLS_cols = []
-for name in painting_image_names:
```

Comment:
Just confirming that you no longer create a figure for all `painting_image_names`. Instead you are creating a single figure, correct?

Reply:
I don't think `painting_image_names` was ever used to make separate figures? Yes, this is a single figure for all the painting images.